homepage.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Yes, I started with the baseline template from Bootstrap and these are part of that. Have to add the CSS and JS links they give also.